srcpkgs/CalculiX-ccx/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'CalculiX-ccx'
+pkgname=CalculiX-ccx
+version=2.20
+revision=1
+archs=~"*-musl"
```

Comment:
why no musl? (here and cgx)

Reply:
Cross-compiling from x86_64 to x86_64-musl failed. Since I could not test this on native x86_64-musl, I don't know if this package works on musl systems.

Reply:
x86_64-musl is not cross compiled, create a musl masterdir instead

Reply:
Oops I missed that part of the manual, thank you very much. It seems to work! I will update the package.

Reply:
PR updated, I removed the "archs" line.